core/vm/runtime/runtime_test.go

```diff
@@ -149,7 +149,7 @@ func BenchmarkCall(b *testing.B) {
 		}
 	}
 }
-func benchmarkEVM_Create(bench *testing.B, code string) {
+func benchmarkevmCreate(bench *testing.B, code string) {
```

Comment:
I think `benchmarkEVMCreate` would be the correct capitalization.

Reply:
Thanks for catching this.